### PR TITLE
Remove chrono quark

### PR DESCRIPTION
**Pull request by talekhinezh**

### CODE REVIEW

File: radixdlt-java/src/main/java/com/radixdlt/client/atommodel/timestamp/TimestampParticle.java [hunk 1]
```diff
@@ -1,22 +1,29 @@
...
 	}
 
 	public TimestampParticle(long timestamp) {
-		super(new ChronoQuark("default", timestamp));
+		this.timestamps.put("default", timestamp);
 	}
 
 	public long getTimestamp() {
-		return getQuarkOrError(ChronoQuark.class).getTimestamp();
+		return timestamps.get("default");
```
Comment (laszlourszuly) [inline radixdlt-java/src/main/java/com/radixdlt/client/atommodel/timestamp/TimestampParticle.java]: Will this change the API (what type of error did `getQuarkOrError` throw)?

Comment (laszlourszuly) [reply radixdlt-java/src/main/java/com/radixdlt/client/atommodel/timestamp/TimestampParticle.java]: Or maybe it doesn't matter...

File: radixdlt-java/src/main/java/com/radixdlt/client/atommodel/timestamp/TimestampParticle.java [hunk 2]
```diff
@@ -1,22 +1,29 @@
...
+	@JsonProperty("timestamps")
+	@DsonOutput(DsonOutput.Output.ALL)
+	private Map<String, Long> timestamps = new HashMap<>();
+
 	private TimestampParticle() {
 	}
 
 	public TimestampParticle(long timestamp) {
-		super(new ChronoQuark("default", timestamp));
+		this.timestamps.put("default", timestamp);
```
Comment (mpsandiford) [inline radixdlt-java/src/main/java/com/radixdlt/client/atommodel/timestamp/TimestampParticle.java]: "default" really needs to be a constant somewhere.

File: radixdlt-java/src/main/java/com/radixdlt/client/atommodel/timestamp/TimestampParticle.java (hunk 1 above)
Comment (mpsandiford) [reply radixdlt-java/src/main/java/com/radixdlt/client/atommodel/timestamp/TimestampParticle.java]: There was really no way for the `TimestampParticle` to be constructed without a `ChronoQuark`, so the "OrError" was not really an actual thing that happened.  The intention was more to signal a programming logic failure of some kind.

Comment (laszlourszuly) [reply radixdlt-java/src/main/java/com/radixdlt/client/atommodel/timestamp/TimestampParticle.java]: 👍

**Comment by Sajjon** [comment]
Too bad that you removed the AtomSerializationTest @talekhinezh , maybe we can re-add an updated one. 

It was there helping out with assuring correctness between libraries - especially helpful to me starting from scratch with the Swift library.

**Pull request merged**